test/test-deride.js

```diff
@@ -360,6 +360,74 @@ describe('Properties', function() {
     });
 });
 
+describe('Exact', function () {
```

Comment:
These should all be inside the `_.forEach` on [line 506](https://github.com/jamesmorgan/deride/blob/ExactMatcher/test/test-deride.js#L506) that way they get tested for each of the different usages.